lib/cli.js

```diff
@@ -244,18 +245,21 @@ export default async () => { // eslint-disable-line complexity
 	const {nonSemVerExperiments: experiments, projectDir} = conf;
 	if (resetCache) {
 		const cacheDir = path.join(projectDir, 'node_modules', '.cache', 'ava');
+
 		try {
-			await del('*', {
-				cwd: cacheDir,
-				nodir: true
-			});
-			console.error(`\n${chalk.green(figures.tick)} Removed AVA cache files in ${cacheDir}`);
+			await fsPromises.access(cacheDir);
+		} catch {
+			console.log(`\n${chalk.green(figures.tick)} No cache files to remove`);
+			process.exit(0); // eslint-disable-line unicorn/no-process-exit
+		}
+
+		try {
+			await del('*', {cwd: cacheDir});
```

Comment:
Looking at the docs for `del`, it returns the paths of the removed files. So we can check if that's empty and update the messaging based on that. I think that's simpler than having a separate check for the directory.

Reply:
Thanks for pointing that out. I can't believe I missed that 😅, I just defaulted to using `fs`. I've made a commit fixing this.